provider/infoblox/infoblox.go

```diff
@@ -546,11 +631,26 @@ func (p *ProviderConfig) recordSet(ep *endpoint.Endpoint, getObject bool, target
 		obj.PtrdName = ep.DNSName
 		obj.Ipv4Addr = ep.Targets[targetIndex]
 		obj.View = p.view
+		sf := map[string]string{
```

Comment:
What does `sf` stand for? `qp` used in ther other parts of the PR looks more intuitive to me.